tests/cnf/core/network/sriov/tests/metricsExporter.go

```diff
@@ -407,6 +408,13 @@ func createTestResources(testRes testResource) *pod.Builder {
 	_, err = testRes.network.Create()
 	Expect(err).ToNot(HaveOccurred(), "Failed to create SR-IOV network")
 
+	By("Verify NAD is created to proceed for Pod creation")
```

Comment:
"Verify NAD is created to proceed with Pod creation"

Reply:
Updated 👍